### PR TITLE
Pid 2289 support jws packer

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

**Comment by vmidyllic** [comment]
do we really need to support signer with bjj on the issuer node?

**Comment by martinsaporiti** [comment]
> do we really need to support signer with bjj on the issuer node?

good catch, thanks

**Review by vmidyllic — APPROVED** [review]

File: cmd/platform/main.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func main() {
 	}
 
 	publisher := gateways.NewPublisher(storage, identityService, claimsService, mtService, keyStore, transactionService, proofService, publisherGateway, networkResolver, ps)
-	packageManager, err := protocol.InitPackageManager(ctx, networkResolver.GetSupportedContracts(), cfg.Circuit.Path)
+	packageManager, err := protocol.InitPackageManager(ctx, networkResolver.GetSupportedContracts(), cfg.Circuit.Path, auth.UniversalDIDResolver)
```
Comment (vmidyllic) [inline cmd/platform/main.go:157]: it is better to provide an ability to configure and pass it both to auth verifier and here, so production will not rely on dev instance of third party project. (as fallback it is ok)

**Review by vmidyllic — CHANGES_REQUESTED** [review]

**Review by x1m3 — APPROVED** [review]
LGTM

File: cmd/platform/main.go (hunk 1 above)
Comment (martinsaporiti) [reply cmd/platform/main.go]: solved

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**